### DISCUSSION TITLE
Can't create request variables

### DISCUSSION THREAD

**Discussion by ghost** [postmanlabs/postman-flows#358] [Q&A] [answered]
Tried, searched, failed.

I have a Send Request block. I am trying to assign one of the attributes from the response body to a variable (aaaa) so that I can use it in the following request block.

I cannot get the "Current Value" to drill into the variables/body to select the "id" attribute in the body.

![image](https://user-images.githubusercontent.com/79308268/182370826-abe507a3-1a65-46e5-850f-59c2eb2e33c8.png)

Any and every selection from the "Current Value" dropdown insists that I complete a boolean expression of some kind via another dropdown:

![image](https://user-images.githubusercontent.com/79308268/182371177-d0741f0a-86f8-4762-b65e-a5df4f2fe0e4.png)

If I manually enter the path to the variable, I get to pick it as either a string or a ref:

![image](https://user-images.githubusercontent.com/79308268/182371668-996eef52-5cd8-450a-adfb-fb7ed3399a2d.png)

Picking either still brings up the expression-builder dropdown. If I cancel the dropdown, I get an error message: "Expected Bool/Number/String/Timestamp, got ∅":

![image](https://user-images.githubusercontent.com/79308268/182372927-59b1645c-a390-471c-9cfa-55e9bc1bca4b.png)

By linking a Terminal block to the output, the only output I can generate is a Boolean result.

I have looked through the examples here and as far as I can tell I am not doing anything different. Is this a bug or am I doing this wrong?

I am using the latest Windows App: Postman v9.26.0 on Windows 10.

**Comment by samay-v** [comment, marked as answer]
Hey @Justus-at-Lextego, 
The issue here is that you don't have an example created for your request. [Here's how to do so](https://github.com/postmanlabs/postman-flows/discussions/358). (Thats what the warning icon ⚠️ next to the request is telling me)

Flows uses Examples to read the response schema, in the absence of it the body is not parsed as JSON, just text. The error is a bug that we're actively working on fixing, you can ignore it for now.

**Reply by ghost** [reply]
Thank you, so I managed to create an example output for all of the endpoints and I am now at least able to create a variable to represent an attribute from within the body.

The next problem I have is that I am trying to merge the outputs of two different requests into an input for a third request. So, request to endpoint A generates a body.id attribute, and endpoint B generates a body.id attribute. How do I get A and B into endpoint C as input parameters linked to variable A and variable B?

OK, figured that out too. Take the output from A and B and combine them into a Create Data block and send them into C from there.
